### ISSUE TITLE
Any Component properties should include button name

### ISSUE THREAD

**Issue by tatiang** [thunkable/thunkable-issues#1085]
**Is your feature request related to a problem? Please describe.**
You should be able to select a button's name (not just its text) in the Any Component drop-down menu (see screenshot below).

**Describe the solution you'd like**

**Describe alternatives you've considered**
Using the button's text but this is less useful than being able to refer to a button by name since it's more likely you'd have something like sequential button names (1, 2, 3, 4) than sequential button text.

**Which platform are you asking for?**
Drag & Drop

**Additional context**
![image](https://user-images.githubusercontent.com/10355498/147376523-c372e781-4653-40f4-b7c8-0a6a21161cb4.png)

**Comment by jaredgibb** [comment]
Agreed! I have added this to our Quality of Life task list! It's not a top priority but it was heard and will be repeated during our upcoming meetings and again in Q1 next year! I encourage you to find others who share in this desire and have them comment here! 

We love to get more input from our users!

**Comment by kezb765** [comment]
I would like to rate this enhancement a top priority. I love to use "any components" but in a complex screen with (let's say) many buttons I alway have a hard time to find out which button has been pressed. To get the name of the component would make my life easier.

**Comment by tatiang** [comment]
I'm still waiting for this to be implemented. It's odd when I'm using Thunkable and certain properties are included (such as the button's text) but not others (in this case, the button's name). I need a way to identify which button the user has clicked. But I typically set the text to "" (blank string) so I can't use that. The button name -- which can be used sequentially as in Thumbnail1, Thumbnail2, etc. -- would be so useful as a condition along with an equals (=) logic block.
